.changeset/lucky-bikes-press.md

```diff
@@ -0,0 +1,7 @@
+---
+'@microsoft/atlas-site': minor
+'@microsoft/atlas-css': major
```

Comment:
[nit] Should this also be `minor` since there aren't any new components, just updates to an existing one?